client/src/components/application/Transactions.vue

```diff
@@ -83,7 +92,8 @@
 
           <!-- Items -->
           <div v-else
-            v-for="(transaction, index) in transactionsData"
+            id="transaction-list-item"
```

Comment:
Due to the v-for loop, there will me multiple instances of this element -- use a class name instead of an id.

Reply:
Good call! Will update this